packages/odyssey-react/src/components/Button/Button.stories.tsx

```diff
@@ -81,6 +81,30 @@ Dismiss.argTypes = {
   onClick: { action: 'clicked button/dismiss' },
 };
 
+export const Small = Template.bind({});
+Small.args = {
+  size: "small"
```

Comment:
can we stick to shorthand enums here? `xs` `s` `m` `l` `xl`

We should discuss what the sizing convention needs to be system-wide, but these shorthand enums are common (and brief)

Reply:
Sure! This'll match what we currently use for spacing tokens.

For this case, would you prefer `.is-ods-button-size-s` or `.is-ods-button-s`? 

I think the clarity of the "size" identifier is helpful, but maybe it's not nec.

Reply:
My preference would me omitting the `is-` prefix for this, but given that we use it everywhere we should probably keep it.

My vote is for `.is-ods-button-size-s`

Reply:
Updated!